cg/meta/store/mip_rna.py

```diff
@@ -1,102 +1,133 @@
-        'archive': True,
-    }, {
-        'path': config_data['log_path'],
-        'tags': ['mip-log', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': config_data['sampleinfo_path'],
-        'tags': ['sampleinfo', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': sampleinfo_data['bcftools_merge'],
-        'tags': ['bcftools-combined-vcf', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': sampleinfo_data['multiqc_html'],
-        'tags': ['multiqc-html', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': sampleinfo_data['multiqc_json'],
-        'tags': ['multiqc-json', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': sampleinfo_data['pedigree_path'],
-        'tags': ['pedigree', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': sampleinfo_data['qcmetrics_path'],
-        'tags': ['qcmetrics', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': sampleinfo_data['vep_path'],
-        'tags': ['vep-vcf', 'rd-rna'],
-        'archive': True,
-    }, {
-        'path': sampleinfo_data['version_collect_ar_path'],
-        'tags': ['versions', 'rd-rna'],
-        'archive': True,
-    }]
-
-    for sample_data in sampleinfo_data['samples']:
-        data.append({
-            'path': sample_data['bootstrap_vcf'],
-            'tags': ['bootstrap-vcf', sample_data['id'], 'rd-rna'],
-            'archive': False,
-        })
-        data.append({
-            'path': sample_data['gatk_asereadcounter'],
-            'tags': ['ase-readcounts', sample_data['id'], 'rd-rna'],
-            'archive': False,
-        })
-        data.append({
-            'path': sample_data['gatk_baserecalibration'],
-            'tags': ['bam', 'baserecalibration', sample_data['id'], 'rd-rna'],
-            'archive': False,
-        })
-        data.append({
-            'path': sample_data['gffcompare_ar'],
-            'tags': ['gff-compare-ar', sample_data['id'], 'rd-rna'],
-            'archive': False,
-        })
-        data.append({
-            'path': sample_data['mark_duplicates'],
-            'tags': ['mark-duplicates', sample_data['id'], 'rd-rna'],
-            'archive': False,
-        })
-        data.append({
-            'path': sample_data['salmon_quant'],
-            'tags': ['salmon-quant', sample_data['id'], 'rd-rna'],
-            'archive': False,
-        })
-        data.append({
-            'path': sample_data['star_fusion'],
-            'tags': ['star-fusion', sample_data['id'], 'rd-rna'],
-            'archive': False,
-        })
-        data.append({
-            'path': sample_data['stringtie_ar'],
-            'tags': ['stringtie-ar', sample_data['id'], 'rd-rna'],
-            'archive': False,
-            })
+    data = [
+        {"path": file["path"], "tags": get_tags(file), "archive": False}
+        for file in deliverables["files"]
+    ]
 
     return data
+
+
+def get_tags(file: dict) -> list:
```

Comment:
I think this one should be in store/base.py for easy reuse by other pipelines. This should be agnostic to all workflows using the deliverables files right?

Reply:
add parameter pipeline  to enable this to be agnostic

Reply:
Moved to `base.py` and made pipeline agnostic.